plugins/org.locationtech.udig.ui/src/org/locationtech/udig/ui/graphics/SWTGraphics.java

```diff
@@ -353,12 +353,14 @@ public void setTransform( AffineTransform transform ) {
         transform.getMatrix(matrix);
         if( swtTransform==null ){
             swtTransform=new Transform(display, 
-                    (float)matrix[0], (float)matrix[1], (float)matrix[2], 
-                    (float)matrix[3], (float)matrix[4], (float)matrix[5] );
+                    (float)matrix[0], (float)matrix[2],
+                    (float)matrix[1], (float)matrix[3],
+                    (float)matrix[4], (float)matrix[5] );
         }else{
             swtTransform.setElements(
-                    (float)matrix[0], (float)matrix[1], (float)matrix[2], 
-                    (float)matrix[3], (float)matrix[4], (float)matrix[5] );
+                    (float)matrix[0], (float)matrix[2],
```

Comment:
looks like duplicated code, What do you think about this:
```
        if( swtTransform==null ){
            swtTransform=new Transform(display); 
        }
        swtTransform.setElements(
                    (float)matrix[0], (float)matrix[2], 
                    (float)matrix[1], (float)matrix[3],
                    (float)matrix[4], (float)matrix[5]);
```

Reply:
Love it.

Reply:
Refactored. The null check is gone too.